app/templates/index.hbs

```diff
@@ -31,6 +31,9 @@
           <c.header>
             <AuDataTableThSortable @label='Naam' @field='label' @currentSorting={{this.sort}}
               class="au-u-visible-medium-up" />
+            <AuDataTableThSortable @label='Beschrijving' @field='comment' @currentSorting={{this.sort}}
+              class="au-u-visible-medium-up" />
+
             <AuDataTableThSortable @label='Aangemaakt op' @field='created' @currentSorting={{this.sort}}
               class="au-u-visible-small-up" />
             <AuDataTableThSortable @label='Aangepast op' @field='modified' @currentSorting={{this.sort}}
```

Comment:
This is an component that cannot read the translations. We could change it to a normal table as we did for the codelists